addons/rose/tests/integration/components/rose/form/fieldset-test.js

```diff
@@ -0,0 +1,79 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, find } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | rose/form/fieldset', function (hooks) {
+  setupRenderingTest(hooks);
+
+  // Global values for mocking
+  this.mockTitle = 'Title';
+  this.mockDescription = 'Description';
+  this.mockBody = 'Rest of the form';
+
+  // Wrap the rendering in a function to avoid repeat code
+  async function renderComponent(noDescription) {
+    if (noDescription === true) {
+      return render(hbs`
+        <Rose::Form::Fieldset>
+          <:title>{{this.mockTitle}}</:title>
+          <:body>{{this.mockBody}}</:body>
+        </Rose::Form::Fieldset>
+    `);
+    }
```

Comment:
Wondering if `renderComponent` should always render the full component (without any conditionals) and the test for missing description should define it's own template without using `renderComponent`. This approach would clarify why missing description block is not a normal state. Thoughts?

Reply:
I see it as description is an optional element of the fieldset. If is provided awesome, but if is not still a normal state (IMHO).
I create the function because I was repeating code in each test case, and if you check the last changes on the tests we will be repeating code even more since I just add a new test for aria when description is not provided.
But happy to discuss and change the approach.

Reply:
If only we had a standardized way to document component inputs.  :sigh:  This part of Ember still feels a little rough.